server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/StatusCode.scala

```diff
@@ -0,0 +1,8 @@
+package org.apache.james.jmap.model
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Interval.Closed
+
+object StatusCode {
+  type ErrorStatus = Int Refined Closed[100, 511]
```

Comment:
I think you can go until 599: https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml